whitepapers/Tables.md

```diff
@@ -1,5 +1,106 @@
 # SQL Tables
 
+## Template
 
+### {ModelName}
+{Description}
 
+#### Columns
+* `{column_name}` -- {description} ({datatype}, [unique,] [indexed,] [nullable,] [default={value},] [choices={choices],])
```

Comment:
| Column | Type | Indexed | Nullable | FK | Default | Description |
|:--------|:------:|:-------:|:--------:|:---:|:---------:|:-------------|
| suid_id |  int  |    X    |          |   X    |             | Description as the end because it may be very very long |
| suid_id |  int  |    Yes    |  No        |   Yes    |    No         | Description as the end because it may be very very long |
| suid_id |  int  |   ✅   |  ❌        |   ✅    |   ❌         | Description as the end because it may be very very long |
| suid_id |  int  |    ✓    |  ⨉        |   ✓    |    ⨉         | Description as the end because it may be very very long |

?